lib/std/net.zig

```diff
@@ -314,20 +338,35 @@ pub const IpAddress = extern union {
                 }
                 try std.fmt.format(context, Errors, output, "]:{}", port);
             },
+            os.AF_UNIX => {
+                if (@typeOf(self.un) == void) {
+                    @panic("unsupported OS");
+                }
```

Comment:
I think `assert` or `unreachable` is more appropriate here. It's definitely the programmer abusing the API if this is reached. Same thing in `getOsSockLen`.